webrender/src/renderer.rs

```diff
@@ -691,6 +696,15 @@ impl Renderer {
         *notifier_arc = Some(notifier);
     }
 
+    /// Sets the new MainThreadDispatcher.
+    ///
+    /// Used to dispatch functions to the main thread's event loop.
+    /// Required to allow GLContext sharing in some implementations like WGL.
```

Comment:
probably moving the last part of this comment to the `main_thread_dispatcher` member definition would make it a bit more helpful.